bundle/manifests/patterns-operator.clusterserviceversion.yaml

```diff
@@ -176,6 +189,17 @@ spec:
           verbs:
           - get
           - list
+        - apiGroups:
+          - ""
+          resources:
+          - secrets
+          verbs:
+          - create
+          - get
+          - list
+          - patch
```

Comment:
Definitiely these too?
Maybe get and create/update cover the functionality of list/patch

Reply:
I can try and remove those and see if we can avoid them.